src/lib/stitching/mergeSchemas.ts

```diff
@@ -78,11 +78,6 @@ export const incrementalMergeSchemas = (localSchema, testConfig?: any) => {
     resolvers: extensionResolvers,
   })
 
-  // Because __allowedLegacyNames isn't in the public API
-  Object.defineProperty(mergedSchema, "__allowedLegacyNames", {
-    value: ["__id"],
-  })
-
```

Comment:
Don’t we still need this for v1, though?

Reply:
🤦‍♂ 

You're right. I'll change this to just have the thing above.

Reply:
Hah. Great reminder that we can remove the graphql-js patch from Emission, though!